spec/ParseGraphQLServer.spec.js

```diff
@@ -9344,6 +9377,51 @@ describe('ParseGraphQLServer', () => {
           }
         });
 
+        it('should not upload if file is too large', async () => {
```

Comment:
Is it a breaking change?

Reply:
No, a missing test, from the previous GQL API, the max file size was not tested on Upload. I added the test.